docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-merge-pr.md

```diff
@@ -60,3 +76,9 @@ steps:
     prNumber: 42
     wait: true
 ```
+
+:::note
+If the repository requires using a merge queue and the token lacks bypass permissions,
+the step will fail regardless of the `wait` setting, as the step does not add PRs to
+merge queues.
+:::
```

Comment:
Upon further thought, I think it's better to just get rid of this. We already said enough about behavior when a merge queue is required.

```suggestion
```